IBSWeb/Areas/Filpride/Controllers/ReportController.cs

```diff
@@ -3054,196 +3086,601 @@ public async Task<IActionResult> GeneratePurchaseReportExcelFile(ViewModelBook m
                 #region -- Initialize "total" Variables for operations --
 
                 var totalVolume = purchaseReport.Sum(pr => pr.QuantityReceived);
-                decimal totalCostPerLiter = 0;
-                decimal totalCostAmount = 0;
-                decimal totalVatAmount = 0;
-                decimal totalWHTAmount = 0;
-                decimal totalNetPurchases = 0;
+                decimal? totalCostPerLiter = 0m;
```

Comment:
Find a way to lessen the initialization like this, the decimal should be not nullable because the default value of this type was zero